core/src/test/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityTest.java

```diff
@@ -85,7 +157,7 @@ public void testBasicBackwardsCompatibility() {
                checker.isCompatible(schema2, Collections.singletonList(schema1)));
     assertFalse("adding a field w/o default is not a backward compatible change",
                 checker.isCompatible(schema3, Collections.singletonList(schema1)));
-    assertFalse("changing field name is not a backward compatible change",
+    assertTrue("changing field name with a proper alias is a backward compatible change",
```

Comment:
@gwenshap and @ewencp - I've started to investigate avro, cp-schema-registry and how schema evolution works.
I'm also running into questions regarding using `alias` in an avro schema. I found the following conversation, which looks to be the thoughts around not supporting alias: https://groups.google.com/forum/#!topic/confluent-platform/0NlrxFD5FHk 

- I'm guessing this will not change?
- Any thoughts on alias support as a user option/configuration?